lib/tpm2_auth_util.c

```diff
@@ -5,6 +5,8 @@
 #include <stdbool.h>
 #include <stdlib.h>
 #include <string.h>
```

Comment:
this block is for std headers, after this we would be system specific libraries (like unistd.h) followed by 3rd party headers followed by internal header files.

Why do you need termios? What benefits does it give us over fread(stdin)?

Reply:
Oh I see what you mean:
https://stackoverflow.com/questions/6856635/hide-password-input-on-terminal